MITosL: a novel approach to identifying and ranking
critical non-proximal interdependencies within the
overall protein structure

The prediction of protein structures is a critical endeavor in bioinformatics and molecular biology. These predictions play a pivotal role in understanding protein function, drug discovery, and disease mechanisms. Despite notable progress, challenges persist in accurately capturing intricate details, especially regarding non-proximal interdependencies and dynamic behaviors. Ongoing research focuses on refining computational approaches, incorporating diverse data sources, and enhancing the precision of predictions for a deeper comprehension of protein structure and function. Advancements in computational methods, such as the groundbreaking AlphaFold, have significantly improved our ability to forecast the three-dimensional arrangement of amino acids in a protein.
The analysis is based of bioinformatics journal “PSICalc: a novel approach to identifying and ranking critical non-proximal interdependencies within the overall protein structure”, Townsley T D, Wilson J T, Akers H, Bryant T, Cordova S, Wallace T L, Durston K K, Deweese J E, 2022. The tool employs pattern-discovery techniques to analyze extensive Multiple Sequence Alignments (MSAs) of both structured and unstructured proteins. From the study of PSICalc, the method used in this paper similarly identifies proximal and non-proximal interdependent sites, clustering them into pairwise (second order), third order, and higher-order clusters using a k-means approach. We illustrate the tool's capabilities by presenting examples of both proximal and non-proximal interdependencies identified in eukaryotic topoisomerase II.
![image](https://github.com/msnerd1312/top2a-mitos-mi/assets/83843924/311614fc-1e99-4575-a79b-cb6b3d688118)
